about.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Capstone 1</title>
+    <link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" />
+    <link rel="stylesheet" href="css/index.css" />
+    <script
+      src="https://kit.fontawesome.com/7415ea4535.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <!---------------- Navigation ------------->
+    <div id="top-div" class="desktop">
+      <span class="fa fa-twitter"></span>
+      <span class="fa fa-facebook"></span>
+      <span>English</span>
+      <span>My Page</span>
+      <span>Logout</span>
+    </div>
+    <nav id="topbar" class="d-flex">
+      <a href="#" class="mobile fa fa-bars" id="mobile-menu-icon"> </a>
+      <div class="desktop">
+        <div class="menu-left">
+          <a href="index.html"><img src="/images/logo-c.png" alt="" /></a>
+        </div>
+        <div class="menu-right">
+          <ul>
+            <li><a href="about.html" class="nav-item">About</a></li>
+            <li><a href="#" class="nav-item">Program</a></li>
+            <li><a href="#" class="nav-item">Join</a></li>
+            <li><a href="#" class="nav-item">Sponsor</a></li>
+            <li><a href="#" class="nav-item">News</a></li>
+            <li><a href="#" class="nav-item campaign">Campaign</a></li>
```

Comment:
- [ ] Honestly your design is incredible :100: , however you just need to improve some little thing to improve the user experience and to meet the requirements :+1: . It could be better if these nav links provided in your `home` and `about` pages are responsible, it means :m: once the user click on them it should bring him to the corresponding section

### What to do ?
Kindly add the section ID in the `href` attribute of the nav `a`
```bash
    <li><a href="section-id" class="nav-item">Program</a></li>
```

![Screenshot from 2023-02-09 22-55-16](https://user-images.githubusercontent.com/80458199/217942902-bb183504-51f4-4d01-8b21-1577015da92a.png)